frontend/src/lib/utils.js

```diff
@@ -61,6 +61,16 @@ export const getPotionColorByDifficulty = (difficulty) => {
   }
 };
 
+export const getBookColor = {
+  "harry-potter-and-the-philosopher-s-stone": "#140c36",
+  "harry-potter-and-the-chamber-of-secrets": "#518f4c",
+  "harry-potter-and-the-prisoner-of-azkaban": "#1b4993",
+  "harry-potter-and-the-goblet-of-fire": "#73aac9",
+  "harry-potter-and-the-order-of-the-phoenix": "#b43223",
+  "harry-potter-and-the-half-blood-prince": "#040907",
+  "harry-potter-and-the-deathly-hallows": "#51476a"
```

Comment:
Can you change the colors for the 1st and 6th book? Those are really hard to see. Maybe a bit lighter version of the 2 colors are better

Reply:
And can you also put the whole method in a switch statement? Wwith a default color, if the slug doesn't match.

Reply:
And don't forget to change it in other places, too.